spaceship/lib/spaceship/tunes/build.rb

```diff
@@ -173,29 +174,11 @@ def submit_for_beta_review!(metadata)
           build_number: self.build_version,
           platform: self.platform,
 
-          # Required Metadata:
-          changelog: "No changelog provided",
```

Comment:
Where do we document the available values? I think that's what it was initially here for